ramp-frontend/ramp_frontend/templates/teaching.html

```diff
@@ -27,27 +27,107 @@ <h4 class="modal-title" id="myModalLabel">{{ category }}</h4>
+                        building blocks from <a href="https://github.com/paris-saclay-cds/ramp-workflow">
+                        RAMP workflow</a>.
+                    </li>
+                    <li>
+                        The data, split into training and test subsets.
+                    </li>
+                    <li>
+                        A requirements file detailing all the required software
+                        packages.
+                    </li>
+                </ul>
+            </p>
+            <p>
+                A challenge event can now be organised. These events may
+                be part of a post-graduate level programme or a 'hackathon'.
+                If the data science problem requires the mastering
+                of a specific tool, the RAMP event can be preceded by a
+                Training Sprint for explaining specific tools to the participants.
+            </p>
+
+            <p>
+                RAMP data challenges has been used for teaching in many courses
+                at various levels. Students usually love the platform for:
+                <ul>
+                    <li>its competitive gaming aspect,</li>
+                    <li>being able to learn from each other in the open phase, and</li>
+                    <li>to compete as a group with other student groups on the same problem.</li>
+                </ul>
+            </p>
+
+            <br>
+
+            <p>
+                <h1 style='font-weight: bold;'>Using RAMP for teaching</h1>
+                If you are interested in using the platform for teaching,
+                We can open a new event for any <a href="/problems">
+                RAMP challenge</a>. You will have full control over the users
+                who can sign up for your event and the timeline the event
+                (start, moving from closed to open phase and end).
+            </p>
+
+            <p>
+                At the end of your challenge we will give you a csv file with all
+                the performance metrics of each student.
+            </p>
+
+
+            <br>
+
+            <p>
+                <h1 style='font-weight: bold;'>Creating a new RAMP challenge</h1>
+                If you have a predictive problem, you can work with us to
+                develop a new RAMP challenge to incite data scientists to
+                collaboratively prototype new models. We can then organize
+                hackathons or longer data challenges, and use the problem in a
+                classroom setting. We can also automatically benchmark the
+                thousands of models that are already in the platform.
+            </p>
+
+            <br>
+
+            <p>
+                <h1 style='font-weight: bold;'>Contact us</h1>
+
+                To use RAMP for teaching or to work with us to develop a new
+                RAMP challenge, contact us at <a href="mailto:admin@ramp.studio"><strong>admin@ramp.studio</strong></a>.
```

Comment:
I would also that RAMP code is open source so you can easily set up your own server. That's the easiest way to make your own challenge. It does not require our time